src/main/java/LoggingExample.java

```diff
@@ -0,0 +1,24 @@
+import java.util.logging.Logger;
```

Comment:
still need this file?

Reply:
Yes, we will need this to copy from it. I will delete it in the next two weeks :)

Reply:
trust you. dont forget